brevent/src/main/res/values/strings.xml

```diff
@@ -180,18 +180,18 @@
     <string name="brevent_language_auto">Auto</string>
     <string name="brevent_theme">Theme</string>
     <string name="show_framework_apps">Show core apps</string>
-    <string name="show_framework_apps_label">All apps have the same signature with framework</string>
+    <string name="show_framework_apps_label">All framework apps have the same signature</string>
```

Comment:
This doesn't explain framework apps.

Framework apps are a category in Brevent, it means apps have the same signature with Android Framework (with the package name `android`).

Reply:
Oh ok, I got it. I have changed it to an understandable way now.